src/Mutation.php

```diff
@@ -132,4 +149,12 @@ public function isCoveredByTest(): bool
     {
         return $this->isCoveredByTest;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getMutatedNode()
+    {
+        return $this->mutatedNode;
```

Comment:
how can the node be an `int`, `array` or `null`?

Reply:
see https://github.com/infection/infection/pull/450#discussion_r209751722

in PHP-Parser API, you can replace node with another node, with array of nodes, you can return int like `NodeTraverser::STOP_TRAVERSAL`, etc.